code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -61,8 +61,36 @@
 	new /obj/item/weapon/weldingtool(src)
 	new /obj/item/weapon/crowbar(src)
 	new /obj/item/weapon/wirecutters(src)
-	new /obj/item/stack/cable_coil(src,30,pick("red","yellow","orange"))
+	new /obj/item/stack/cable_coil(src,30,pick("red","yellow","green","blue","pink","orange","cyan","white"))

```

Comment:
If memory serves, should you not supply any color to coils at all they'll select a valid random color for you.